product-list-BE/server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const router = require('./routes/main'); 
+const cors = require('cors');
+
+const app = express();
+const PORT = process.env.PORT || 8000;
+
+// Middleware
+app.use(cors());
+app.use(bodyParser.json());
+
+// Connect to MongoDB
+mongoose.connect('mongodb://127.0.0.1:27017/Products') 
```

Comment:
Important, use lowercase for the DB. Mongo is case sensitive. 
```suggestion
mongoose.connect('mongodb://127.0.0.1:27017/products') 
```